locales/pl.json

```diff
@@ -1,155 +1,164 @@
-	"To continue supporting this growth, and to satisfy community expectations,": "To continue supporting this growth, and to satisfy community expectations,",
-	"we would like to invest more time and resources in this project.": "we would like to invest more time and resources in this project.",
-	"If you like this project and see its potential, you can help by supporting us on OpenCollective!": "If you like this project and see its potential, you can help by supporting us on OpenCollective!",
-	"Thanks,": "Thanks,",
-	"Boostnote maintainers": "Boostnote maintainers",
-	"Support via OpenCollective": "Support via OpenCollective",
-	"Language": "Language",
-	"English": "English",
-	"German": "German",
-	"French": "French",
-	"Show \"Saved to Clipboard\" notification when copying": "Show \"Saved to Clipboard\" notification when copying",
-	"All Notes": "All Notes",
-	"Starred": "Starred",
-	"Are you sure to ": "Are you sure to ",
-	" delete": " delete",
-	"this folder?": "this folder?",
-	"Confirm": "Confirm",
-	"Cancel": "Cancel",
-	"Markdown Note": "Markdown Note",
-	"This format is for creating text documents. Checklists, code blocks and Latex blocks are available.": "This format is for creating text documents. Checklists, code blocks and Latex blocks are available.",
-	"Snippet Note": "Snippet Note",
-	"This format is for creating code snippets. Multiple snippets can be grouped into a single note.": "This format is for creating code snippets. Multiple snippets can be grouped into a single note.",
-	"Tab to switch format": "Tab to switch format",
-	"Updated": "Updated",
-	"Created": "Created",
-	"Alphabetically": "Alphabetically",
-	"Default View": "Default View",
-	"Compressed View": "Compressed View",
-	"Search": "Search",
-	"Blog Type": "Blog Type",
-	"Blog Address": "Blog Address",
-	"Save": "Save",
-	"Auth": "Auth",
-	"Authentication Method": "Authentication Method",
+	"Dear everyone,": "Droga społeczności,",
+	"Thank you for using Boostnote!": "Dziękujemy za używanie Boostnote!",
+	"Boostnote is used in about 200 different countries and regions by an awesome community of developers.": "Boostnote jest używany w około 200 krajach i regionach przez wspaniałą społeczność programistów.",
+	"To continue supporting this growth, and to satisfy community expectations,": "Chcielibyśmy poświęcić więcej czasu na rozwój naszego projektu",
+	"we would like to invest more time and resources in this project.": "aby popularność i satysfakcja naszej społeczności ciągle wzrastała.",
+	"If you like this project and see its potential, you can help by supporting us on OpenCollective!": "Jeśli podoba Ci się naszy projekt i lubisz go używać, możesz wspomóc nasz przez OpenCollective!",
+	"Thanks,": "Dzięki,",
+	"Boostnote maintainers": "Kontrybutorzy Boostnote",
+	"Support via OpenCollective": "Wspomóż przez OpenCollective",
+	"Language": "Język",
+	"English": "Angielski",
```

Comment:
I think you should not change the language name because if someone "accidentally" change the language to Polish, it's really hard for them to find their desired language if they don't speak Polish :smile:

Reply:
This is the same case when you accidentally change the language on your smartphone. And they don't cover this case, you need to reset your device. IMO we should add the button at the bottom of the preferences with label `default settings` or something similar

Reply:
Plus the application has support for only a few languages (Some of them aren't translated so they got English translations), so there won't be the problem to find the correct language

Reply:
Hmm... I actually agree with @ZeroX-DG. It should be terrible if users need to lose whole settings just because they set a wrong language. I'll fix this after I merging this PR.